README.md

```diff
@@ -1,39 +1,56 @@
-# Helix
+<h1 align="center" style="font-size: 55px">Helix</h1>
+<p align="center" style="font-size: 30px">A kakoune / neovim inspired editor, written in Rust.</p>
 
+<div align="center" style="display:inline">
+      <img src="screenshot1.png" width="49%">
+      <img src="screenshot2.png" width="49%">
+</div>
 
-[![Build status](https://github.com/helix-editor/helix/actions/workflows/build.yml/badge.svg)](https://github.com/helix-editor/helix/actions)
-
-![Screenshot](./screenshot.png)
 
-A kakoune / neovim inspired editor, written in Rust.
+[![Build status](https://github.com/helix-editor/helix/actions/workflows/build.yml/badge.svg)](https://github.com/helix-editor/helix/actions)
+[![Matrix](https://img.shields.io/matrix/helix-community:matrix.org)](https://img.shields.io/matrix/helix-community:matrix.org)
+[![License](https://img.shields.io/github/license/helix-editor/helix)](./LICENSE)
 
 The editing model is very heavily based on kakoune; during development I found
-myself agreeing with most of kakoune's design decisions.
-
-For more information, see the [website](https://helix-editor.com) or
+myself agreeing with most of kakoune's design decisions. For more information, see the [website](https://helix-editor.com) or
 [documentation](https://docs.helix-editor.com/).
 
-All shortcuts/keymaps can be found [in the documentation on the website](https://docs.helix-editor.com/keymap.html).
-
-[Troubleshooting](https://github.com/helix-editor/helix/wiki/Troubleshooting)
+All shortcuts/keymaps can be found [on the website](https://docs.helix-editor.com/keymap.html).
 
-# Features
+## ✨ Features
 
 - Vim-like modal editing
 - Multiple selections
 - Built-in language server support
 - Smart, incremental syntax highlighting and code editing via tree-sitter
 
 It's a terminal-based editor first, but I'd like to explore a custom renderer
-(similar to emacs) in wgpu or skulpin.
+(similar to emacs) in wgpu or skulpin. See [#39](https://github.com/helix-editor/helix/issues/39) for more info.
+
+## 📦 Installation
+
+> Note: Installing from source requires extra configuration for syntax highlighting and themes to work. See [runtime](#runtime).
+
+#### Binary (All Platforms)
+
+Pre-build binaries are available in the [releases page](https://github.com/helix-editor/helix/releases).
+
+#### Linux
 
-# Installation
+Download via a package:
 
-Note: Only certain languages have indentation definitions at the moment. Check
-`runtime/queries/<lang>/` for `indents.toml`.
+[![Packaging status](https://repology.org/badge/vertical-allrepos/helix.svg)](https://repology.org/project/helix/versions)
```

Comment:
I'd rather list this on top (under binary packages) since repology will list all available packages, including windows and macOS (homebrew)

Reply:
Is this what you mean? #4b98858

Reply:
Yep 👍🏻